src/main/java/appeng/core/features/registries/cell/BasicCellHandler.java

```diff
@@ -47,23 +50,34 @@
 	@Override
 	public boolean isCell( final ItemStack is )
 	{
-		return CellInventory.isCell( is );
+		return ItemCellInventory.isCell( is ) || FluidCellInventory.isCell( is );
```

Comment:
It might be an idea to split it into two handler. Similar to the item and creative handler.

Reply:
maybe we can add an api to add a default cell inventory for IStorageCell of each channel. We can check if the item is a IStorageCell and then use the registerd CellInventory

Reply:
Already exists with `ICellRegistry#getHandler(ItemStack is);`